CONTRIBUTING.markdown

```diff
@@ -1,114 +1,183 @@
-a command line tool for issuing pull requests, developed specifically for the
-Liferay Community.
-* In the LPS ticket, provide a link to your GitHub pull request and respond to
-the [Contributor License
-Agreement](http://www.liferay.com/legal/contributors-agreement) displayed by
-clicking the *Contribute Solution* button.
-* You're done! Well, not quite ... be sure to respond to comments and questions
-to your pull request until it is closed.
-
-## Additional Resources
-
-* [Getting Started as a Liferay
-Developer](http://www.liferay.com/community/wiki/-/wiki/Main/Getting+started+as+a+Liferay+Developer+in+a+few+steps)
-* [Optimal Liferay Core
-Development](http://www.liferay.com/community/wiki/-/wiki/Main/Optimal+Liferay+Core+Development)
-* [Liferay and JIRA](http://www.liferay.com/community/wiki/-/wiki/Main/JIRA)
-* [Contribute to Liferay on
-Github](http://www.liferay.com/community/wiki/-/wiki/Main/Contribute+using+Git+and+GitHub)
-* [Liferay Core Development
-Guidelines](http://www.liferay.com/community/wiki/-/wiki/Main/Liferay+Core+Development+Guidelines)
-* [Setting up and using Liferay
-IDE](http://www.liferay.com/community/wiki/-/wiki/Main/Liferay+Contributor+Development+Environment+Setup)
-* [Contributor License
-Agreement](http://www.liferay.com/legal/contributors-agreement)
-* [General GitHub documentation](http://help.github.com/)
-* [GitHub pull request
-documentation](http://help.github.com/send-pull-requests/)
-* [Liferay's IRC channel on
-freenode.org](http://webchat.freenode.net/?channels=liferay&uio=d4)
\ No newline at end of file
+
+# Liferay Contribution Process
+
+Liferay 7 went from being a single monolithic application to being composed of many modules. This has lead to changes in the way that we develop for Liferay as well as how we accept contributions from the community.
+
+## Liferay Central vs. Sub Repos
+
+The Liferay organization on Github has many new repos that start with com-liferay-* and well as the traditional Liferay Central (**liferay-portal**) repos.
+
+With modularity comes interesting opportunities for increasing developer productivity. One large change we are working on is the addition of autonomous Sub Repos. Sub Repos allow for independently developing sections of the platform and then merging those changes back into a central repository.
+
+Ultimately, all source will reside in a Sub Repo and all contributions would take place in the Sub Repo.  Depending on the type of contribution, one of three areas may be involved:
+
+* **Platform Core** - These are parts of the original platform that have not been extracted to a module as of yet. They carry familiar names from Liferay 6.2 such as: portal-impl, portal-kernel and portal-web and reside in the Liferay Central repo.
+
+* **Modules in Central** - These modules are maintained within the Liferay Central repo, however a Sub Repo exists and are in “push” mode meaning the changes to the module in liferay-portal are propagated to the Sub Repo.
```

Comment:
I would skip this since it's hard to understand why we do this (there is no good reason) and it's not really necessary for them to know.